modules/ols-openshift-requirements.adoc

```diff
@@ -0,0 +1,8 @@
+:_mod-docs-content-type: CONCEPT
+[id="ols-openshift-requirements"]
+= OpenShift Requirements 
+:context: ols-openshift-requirements
+
+{ols-long} requires {ocp-product-title} 4.15 or later running on x86 hardware. Any installation type or deployment architecture is supported so long as the cluster is 4.15+ and x86-based.
+
+For the {ols-long} {ols-release} release, the cluster you use must be connected to the Internet and it must have telemetry enabled. Telemetry is enabled by default, so please confirm that if you are using a standard installation process for {ocp-short-name} that it does not disable telemetry.
```

Comment:
For the {ols-long} {ols-release} release, the cluster you use must be connected to the Internet and it must have telemetry enabled. Telemetry is enabled by default, so please confirm that, if you are using a standard installation process for {ocp-short-name}, it does not disable telemetry.    maybe?

Reply:
@gabriel-rh The sentence is long so I ended up just breaking it like so:

Telemetry is enabled by default. If you are using a standard installation process for {ocp-short-name} confirm that it does not disable telemetry.